documentation/IDTA-01002-3/modules/ROOT/pages/Annex/IDTA-01002_SerializationModifierExamples.adoc

```diff
@@ -289,12 +319,22 @@ If applied to the Submodel:
 
 ----
 
+If applied to the SubmodelElementCollection:
+
+[source,json,linenums]
+----
+[
+  "RotationSpeed.MaxRotationSpeed"
+]
+----
 
 If applied to the Property inside the SubmodelElementCollection:
 
 [source,json,linenums]
 ----
-[ ]
```

Comment:
Since property has no subelement it seems correct to me to return empty set? Or just return MaxRotationSpeed - but for collection you also do not return reference to collection only, but for its elements only? For Collection you also do not mention the submodel the collection is part of.

Reply:
> but for collection you also do not return reference to collection only, but for its elements only?

Yes, as "core" says the thing itself (here the SubmodelElementCollection) + its direct children (the Property). If there were even more levels (additional SubmodelElementCollections or SubmodelElementLists), their elements would be skipped. But here we only have one.

> For Collection you also do not mention the submodel the collection is part of.

Correct, as the IdShortPath needs to start at the first SubmodelElement, not containing the idShort of the Submodel at all.